appmonitoring/ts/src/CertificateGenerator.ts

```diff
@@ -0,0 +1,229 @@
+import * as k8s from "@kubernetes/client-node";
+import forge from "node-forge";
+import pem from "pem";
+
+class WebhookCertData {
+    caCert: string;
+    tlsCert: string;
+    tlsKey: string;
+}
+
+const hostname = process.env.HOSTNAME;
+
+export class CertificateManager {
+
+    private static makeNumberPositive = (hexString) => {
+        let mostSignificativeHexDigitAsInt = parseInt(hexString[0], 16);
+    
+        if (mostSignificativeHexDigitAsInt < 8) return hexString;
+    
+        mostSignificativeHexDigitAsInt -= 8
+        return mostSignificativeHexDigitAsInt.toString() + hexString.substring(1)
+    }
+    
+    // Generate a random serial number for the Certificate
+    private static randomSerialNumber = () => {
+        return CertificateManager.makeNumberPositive(forge.util.bytesToHex(forge.random.getBytesSync(20)));
```

Comment:
Figure out a way to generate a random positive number, then convert it to a HEX string